src/rtk/features/apiDelegation/overviewOrg/overviewOrgSlice.ts

```diff
@@ -321,22 +307,26 @@ const overviewOrgSlice = createSlice({
   extraReducers: (builder) => {
     builder
       .addCase(fetchOverviewOrgsReceived.fulfilled, (state, action) => {
+        state.loading = true;
```

Comment:
Why set this here? 
The loading should be set before the call to the API happens (since that is the part that will take time)